model/setup.sql

```diff
@@ -0,0 +1,105 @@
+CREATE SCHEMA tournamentbuzz;
+USE tournamentbuzz;
+
+CREATE TABLE users (
+  email VARCHAR(255) NOT NULL UNIQUE CHECK(email REGEXP '^[A-Za-z0-9][A-Za-z0-9]*@[A-Za-z0-9][A-Za-z0-9]*\\.[A-Za-z0-9]*$'),
+  passw VARCHAR(255) NOT NULL CHECK(passw LIKE '%________%'),
+  uname VARCHAR(60),
+  sadmin BOOL DEFAULT FALSE NOT NULL, 
+  PRIMARY KEY(email)
+);
+
+CREATE TABLE tournaments (
+	id INT(10) NOT NULL UNIQUE AUTO_INCREMENT,
+    creator VARCHAR(255) NOT NULL,
+    sponsor VARCHAR(255) DEFAULT NULL,
+    teamEvent BOOL NOT NULL DEFAULT FALSE,
+    location VARCHAR(255) DEFAULT NULL,
+    scoringType ENUM('Points') NOT NULL DEFAULT 'Points',
+    tname VARCHAR(255) DEFAULT NULL,
+    bracketType ENUM('Single Elim', 'Double Elim', 'Round-robin') NOT NULL DEFAULT 'Single Elim',
```

Comment:
Let's just call this tournamenType, since e.g. Round Robin does not involve brackets.